contracts/test/integration/utils/AllArbitrumRoles.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Core, Vcon, Volt, IERC20, IVolt} from "../../../core/Core.sol";
+import {ArbitrumAddresses} from "../fixtures/ArbitrumAddresses.sol";
+import {MainnetAddresses} from "../fixtures/MainnetAddresses.sol";
+import {TribeRoles} from "../../../core/TribeRoles.sol";
+import {DSTest} from "./../../unit/utils/DSTest.sol";
+import {L2Core} from "../../../core/L2Core.sol";
+import {Core} from "../../../core/Core.sol";
+import {Vm} from "./../../unit/utils/Vm.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import {RoleTesting} from "./RoleTesting.sol";
+import {console} from "hardhat/console.sol";
+
+contract ArbitrumTestAllArbitrumRoles is RoleTesting {
```

Comment:
This test structure works and does the job. Personal preference but I like constructing a config for these kind of state tests elsewhere:

```
contract ArbitriumRolesConfig {
  uint256 public constant NUM_GOVERNORS = 2

  bytes32[] public allRoles;
}
```

and then importing the config into the test file. Makes it easy to instantly see the whole state of the roles in one place. 

Then inject the config into the relevant integration/unit test. I would also personally use the full testing function in these integration tests, rather than `_testRoleArity()` use  - although I guess you need that for Arbitrum and Mainnet, so you're making the code more DRY

Reply:
Good call, moved state variables out of the AllNetworkRoles file into a NetworkRolesConfig file for simplicity's sake.